messaging/src/test/java/org/axonframework/eventhandling/tokenstore/jpa/JpaTokenStoreTest.java

```diff
@@ -262,16 +264,72 @@ public void testClaimAndUpdateToken() {
 
     @Transactional
     @Test
-    public void testQuerySegments() {
+    void testFetchTokenBySegment() {
```

Comment:
Nit: I am missing how the tests `testFetchTokenBySegment` and `testFetchTokenBySegmentSegment0` expect the result to not exist.

Reply:
See my reply on the other test in `JdbcTokenStoreTest`, this would be the same case.